llvm/lib/Passes/PassBuilder.cpp

```diff
@@ -821,6 +821,21 @@ Expected<EmbedBitcodeOptions> parseEmbedBitcodePassOptions(StringRef Params) {
   return Result;
 }
 
+Expected<LowerAllowCheckPass::Options>
+parseLowerAllowCheckPassOptions(StringRef Params) {
```

Comment:
if we have parsing, we need 

```
void printPipeline(raw_ostream &OS,
                     function_ref<StringRef(StringRef)> MapClassName2PassName);
```
in the pass

Reply:
Thanks, noted. I will add it in a follow-up patch that performs parsing (and unparsing).